components/cluster/command/clean.go

```diff
@@ -56,6 +58,7 @@ You can retain some nodes and roles data when cleanup the cluster, eg:
 
 	cmd.Flags().StringArrayVar(&cleanOpt.RetainDataNodes, "ignore-node", nil, "Specify the nodes or hosts whose data will be retained")
 	cmd.Flags().StringArrayVar(&cleanOpt.RetainDataRoles, "ignore-role", nil, "Specify the roles whose data will be retained")
+	cmd.Flags().StringArrayVarP(&cleanOpt.CleanDataNodes, "node", "n", nil, "Specify the nodes whose log will be clean")
```

Comment:
1. Why we need this argument to be persistent? 
2. s/Specify the nodes whose log will be clean/Specify the nodes or hosts whose data will be cleaned/

Reply:
One more thing, compared to `--ignore-node` and `--ignore-role`, correspondingly, we also need `--role` parameter.

Reply:
Prefer to keep consistent with existed code:https://github.com/pingcap/tiup/blob/master/components/cluster/command/start.go#L49

```
	cmd.Flags().StringSliceVarP(&gOpt.Roles, "role", "R", nil, "Only clean specified roles")
	cmd.Flags().StringSliceVarP(&gOpt.Nodes, "node", "N", nil, "Only clean specified nodes")
```